data/data/google/bootstrap/main.tf

```diff
@@ -0,0 +1,59 @@
+resource "google_service_account" "cluster" {
+  account_id   = "${var.cluster_name}-cluster"
+  display_name = "Cluster service account"
+}
+
+resource "google_project_iam_member" "cluster" {
+  role    = "roles/editor"
+  member  = "serviceAccount:${google_service_account.cluster.email}"
+}
+
+resource "google_compute_instance" "bootstrap" {
+  name         = "${var.cluster_name}-bootstrap"
+  machine_type = "${var.instance_type}"
+  zone         = "${var.zone}"
+
+  metadata = {
+    user-data = "${var.ignition}"
+  }
+
+  tags = ["ocp", "ocp-master"]
+
+  service_account = {
+    email = "${google_service_account.cluster.email}"
+    scopes = ["compute-rw"]
+  }
+
+  network_interface = {
+    network    = "${var.subnetwork != "" ? "" : var.network}"
+    subnetwork = "${var.subnetwork}"
```

Comment:
Some environments may require using a [GCP Shared VPC](https://cloud.google.com/vpc/docs/shared-vpc). The above code will not work with a Shared VPC.